commands/bridge_pull.go

```diff
@@ -64,14 +74,37 @@ func runBridgePull(cmd *cobra.Command, args []string) error {
 		return nil
 	})
 
-	// TODO: by default import only new events
-	events, err := b.ImportAll(ctx, time.Time{})
+	// initially the importSince time is set the zero value
+	importSince := time.Time{}
+
+	// if noResumeImport is false try to read the last import time from the bridge configuration
+	if !noResumeImport {
+		lastImportTimeStr, err := repo.ReadConfigString(fmt.Sprintf("git-bug.bridge.%s.lastImportTime", b.Name))
```

Comment:
what about a specialized `ReadConfigDate` ? It would simplify the code here and we can easily support more format later (timestamp, data in various formal, duration ...)

Reply:
Good idea that could be very useful